_includes/eipfeedcontent.html

```diff
@@ -0,0 +1,22 @@
+<p><strong>EIP-#{{ eip.eip }}: {{ eip.title }}</strong> is in {{ eip.status }} status. It is authored by {{ eip.author | escape }} and was originally created {{ eip.created | date: '%Y-%m-%d' }}. It is in the {{ eip.category }} category of type {{ eip.type }}.</p>
```

Comment:
Not all categories have types. Can you add an if statement so the type part is omitted if it's not relevant?

Reply:
👍